Scripts/Runtime/Core/Steps/CallMethodAnimationStep.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace BrunoMikoski.AnimationSequencer
+{
+    public class CallMethodAnimationStep : AnimationStepBase
```

Comment:
also its should be a sealed class for now, so `public sealed class CallMethodAnimationStep : AnimationStepBase`